gobrew.go

```diff
@@ -266,9 +268,24 @@ func (gb *GoBrew) ListRemoteVersions(print bool) map[string][]string {
 
 // CurrentVersion get current version from symb link
 func (gb *GoBrew) CurrentVersion() string {
-	fp, err := filepath.EvalSymlinks(gb.currentBinDir)
-	if err != nil {
-		return "None"
+	// https://github.com/golang/go/issues/63703
+	var fp string
+	if runtime.GOOS == "windows" {
+		cmd := fmt.Sprintf(
+			"Get-Item -Path %s | Select-Object -ExpandProperty Target",
+			gb.currentBinDir,
```

Comment:
I need a little more time to check what is offered in
https://github.com/golang/go/issues/63703

I don't quite understand why a call is made specifically ps in this case